src/plugin.js

```diff
@@ -11,7 +12,7 @@ CKEDITOR.plugins.add('uploadcare', {
     editInst.editor = editor;
     var getBody = require('./tools/get-body');
     
-    editor.addContentsCss( this.path + 'styles/plugin.css' );
+    CKEDITOR.addCss(require('./styles/resize'));
```

Comment:
Why css like string in js file? You use browserify for bundle. You also can use https://www.npmjs.com/package/browserify-css for require css file instead js file.